.circleci/config.yml

```diff
@@ -311,42 +339,88 @@ jobs:
 
   test-backend:
     executor: machine-executor
-    parameters:
-
-      owasp:
-        type: boolean
-        default: true
-
-      qasp:
-        type: boolean
-        default: true
-
-    branches:
     steps:
       - checkout
       - docker-compose-check
+      - docker-compose-up-backend
       - run:
-          name: "Test: Build and Spin-up Django API Service"
-          command: "cd tdrs-backend; docker-compose up -d --build"
-      - run:
-          name:  "Test: Execute Python Linting Test"
-          command: "cd tdrs-backend; docker-compose run --rm web bash -c \"flake8 .\""
+          name: Execute Python Linting Test
+          command: cd tdrs-backend; docker-compose run --rm web bash -c "flake8 ."
       - run:
-          name: "Test: Run Unit Tests And Create Code Coverage Report"
+          name: Run Unit Tests And Create Code Coverage Report
           command: |
             cd tdrs-backend;
             docker-compose run --rm web bash -c "./wait_for_services.sh && pytest --cov-report=xml"
       - upload-codecov:
           component: backend
           coverage-report: ./tdrs-backend/coverage.xml
+
+  test-frontend:
```

Comment:
Moved this up so the file is organized more logically. This is the structural pattern it now follows:

version
orbs
executors
commands
-> general commands
-> deployment commands
jobs
-> build-and-test jobs
-> OWASP scan jobs
-> deployment jobs
pipeline parameters
workflows
-> `build-and-test` first since it is the default, then all others alphabetically